requirements.txt

```diff
@@ -1,7 +1,9 @@
+beautifulsoup4==4.9.0
 certifi==2019.11.28
 chardet==3.0.4
 click==7.1.1
 Flask==1.1.1
+html5lib==1.0.1
```

Comment:
`html5lib` is super slow, but we’re not really doing anything high-volume here. Past scraping experience has taught me that `lxml` is fast but not really safe enough for your average web page.

In some other projects, I use [`html5-parser`](https://html5-parser.readthedocs.io/en/latest/), which is a faster version written in C, but installing right requires some fancy pip-specific arguments (so you can no longer use `setup.py install` if you ever want to go down that route). Example: https://github.com/edgi-govdata-archiving/web-monitoring-processing/blob/a05fd623f5b983db84d4f2370197a61bd4276153/requirements.txt#L8-L9

Let me know if I should change.